### PR TITLE
Add Image::GetSizeOfPixelComponent method

### PR DESCRIPTION
**Pull request by blowekamp**
Adding convenience method to get the size of pixel component.

### CODE REVIEW

File: Code/Common/include/sitkImage.h [hunk 1]
```diff
@@ -193,20 +193,27 @@ namespace simple
     /** \brief Get the number of components for each pixel
      *
      * For scalar images this methods returns 1. For vector images the
```
Comment (zivy) [inline Code/Common/include/sitkImage.h:195]: Possibly rephrase:
For images with scalar or complex pixel types this method returns one. For images with vector pixel type the method returns the number of vector components.

File: Code/Common/src/sitkImage.cxx [hunk 2]
```diff
@@ -139,6 +140,64 @@ namespace itk
       return this->m_PimpleImage->GetNumberOfPixels();
     }
 
+  unsigned int Image::GetSizeOfPixelComponent( ) const
+    {
+    switch( this->GetPixelIDValue() )
+      {
+      case sitkUnknown:
```
Comment (zivy) [inline Code/Common/src/sitkImage.cxx:147]: Are both the sitkUnknown and default portions of the switch required? Possibly remove the `case sitkUnknown`?

**Review by zivy — APPROVED** [review]
Minor comments and question.

File: Code/Common/include/sitkImage.h (hunk 1 above)
Comment (blowekamp) [reply Code/Common/include/sitkImage.h]: done

File: Code/Common/src/sitkImage.cxx (hunk 2 above)
Comment (blowekamp) [reply Code/Common/src/sitkImage.cxx]: done

**Pull request merged**